### DISCUSSION TITLE
How to use multiple variants?

### DISCUSSION THREAD

**Discussion by EloB** [motiondivision/motion#1791] [Q&A]
In the documentation they provide an array of strings to the attribute `variants`. Do they mean `animate`, `initial` and `exit`?
https://www.framer.com/docs/animation/##multiple-variants

```jsx
<motion.ul variants={["open", "primary"]} />
```

Like this?
```jsx
<motion.ul variants={myVariants} animate={["open", "primary"]} />
```

**Comment by gianlucajahn** [comment]
Yes, style rules in both animation variants are being applied to the `<ul />` element in your example, although "primary" overwrites "open" if they alter the same values, because the last variant to alter a given value takes precedence.
